docs/docs/dependencies/index.md

```diff
@@ -1,40 +1,78 @@
 
-### Add `@RootBlockEntity()` and `@RootBlock()` decorators to entity
+If you still need to know which entities a block references or in which blocks an entity is used, you can use COMET's block index.
 
-The entity needs to be annotated with `@RootBlockEntity()`.
+---
 
-All fields containing block data need to be annotated with `@RootBlock()`. The Block used by this field must be passed as an argument.
+## Configuration Guide
 
-```ts
-//...
-@RootBlockEntity()
-export class News extends BaseEntity<News, "id"> {
-    // ...
+Follow the upcoming guide if you want to
 
-    @RootBlock(DamImageBlock)
-    @Property({ customType: new RootBlockType(DamImageBlock) })
-    @Field(() => RootBlockDataScalar(DamImageBlock))
-    image: BlockDataInterface;
+-   make the "Dependents" tab in the DAM work (only non-optional steps) or
+-   display the dependencies or dependents of an entity somewhere in your admin app
 
-    @RootBlock(NewsContentBlock)
-    @Property({ customType: new RootBlockType(NewsContentBlock) })
-    @Field(() => RootBlockDataScalar(NewsContentBlock))
-    content: BlockDataInterface;
+### Configuring the block index
 
-    // ...
-}
+#### 1. API: Register fields to be added to the block index
+
+First, you must "tell" the block index which database fields contain block data.
+It will scan these fields for dependency information.
+
+To do that, you must
+
+-   Annotate the entity with `@RootBlockEntity()`
+-   Annotate all columns containing block data with `@RootBlock(ExampleBlock)`
+
+For example:
+
+```diff
++   @RootBlockEntity()
+    export class News extends BaseEntity<News, "id"> {
+        // ...
+
++       @RootBlock(DamImageBlock)
+        @Property({ customType: new RootBlockType(DamImageBlock) })
+        @Field(() => RootBlockDataScalar(DamImageBlock))
+        image: BlockDataInterface;
+
++       @RootBlock(NewsContentBlock)
+        @Property({ customType: new RootBlockType(NewsContentBlock) })
+        @Field(() => RootBlockDataScalar(NewsContentBlock))
+        content: BlockDataInterface;
+
+        // ...
+    }
 ```
 
-## Correctly display a dependency target
+#### 2. Create the block index
+
+You must then create the block index by calling `npm run console createBlockIndexViews` in your `/api` directory.
+This creates a materialized view called `block_index_dependencies` in your database.
 
-### 1. Add `@EntityInfo()` to entity (API)
+You must recreate the block index views after
 
-You can provide the entity info in two ways:
+-   executing database migrations
+-   executing the fixtures (because they drop the whole database and recreate it)
 
-#### `GetEntityInfo` Method
+You should automate the creation by calling `DependenciesService#createViews` in the `MigrateConsole` and `FixturesConsole` of your project.
+Implement this functionality as shown in [this commit](https://github.com/vivid-planet/comet-starter/commit/d6666eb37f7e132a625bcab975cba8676ec6d566).
```

Comment:
This commit could become out-of-date soon, maybe just refer to the migration guide instead? We could also write upgrade scripts that perform the necessary steps?